.jscsrc

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
I tried that, but some of the overrides didn't seem to work, complaining that they could only be turned on, not off. Perhaps I was doing something wrong. I then started by copying the jquery preset, but it also seemed to be missing a ton of rules, so I ended up just going through every rule and picking the ones that applied.